### ISSUE TITLE
some text of SC-IDE for Windows is too small

### ISSUE THREAD

**Issue by prko** [supercollider/supercollider#6154]
## Environment

* SuperCollider version: 3.14.0_dev
* Operating system: Windows 11

## Steps to reproduce
1. Launch SuperCollider
2. Menu -> Edit Preference
    or 
    Look at the text on the "Document tab", "Help Browser" title and "Post window" title. 

## Expected vs. actual behavior
expected:
<img width="2048" alt="Screenshot 2023-12-06 at 16 35 47" src="https://github.com/supercollider/supercollider/assets/416281/d01f750a-7acf-4a7d-9162-55affec135f3">

actual behaviour:
<img width="2018" alt="Screenshot 2023-12-06 at 16 23 21" src="https://github.com/supercollider/supercollider/assets/416281/a7af65e7-21bb-45ca-a97a-1a0c34ee63c2">

**Comment by dyfer** [comment]
@prko I cannot reproduce this on a current dev build on Windows 11. Could you please confirm what resolution(s) and scale settings this occurs at? Does it occur every time? Does it issue persist if the IDE is restarted after the resolution change?

**Comment by prko** [comment]
Yes, it seems to be resolved. I could not reproduce this with a current dev build on Windows 11:

The previously reported image is with a resolution of 2560 * 1440. 

I tested it with a resolution of 2560 * 1440 (100 % font scale) and of 5120 * 2880 (200 % font scale):

![Image](https://github.com/user-attachments/assets/04d9a144-dd97-4544-8b74-6d28515ce102)
![Image](https://github.com/user-attachments/assets/aac0d96d-03df-4268-99a8-e84058f6a486)
